### PR TITLE
Completed 10 UDT's

### PR DESCRIPTION
**Pull request by colliderman**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -53,42 +53,152 @@ struct CarWash            //1) a U.D.T. with a random number of member variables
...
+    bool startButtonHeld = false;
+
+    struct Time
+    {
+        unsigned int seconds, minutes, hours;
+    };
+
+    void startTimer( Time* time );
+    void stopTimer( Time* time );
+    void resetTimer( Time* time );
```
Comment (matkatmusic) [inline main.cpp]: It's too soon for pointers.  keep it simple, add a `Time` class member, and remove these function parameters.  the next project part has you implement these functions, where you'll interact with this `Time` member.

- [ ] Keep It Simple for now.

File: main.cpp [hunk 2]
```diff
@@ -53,42 +53,152 @@ struct CarWash            //1) a U.D.T. with a random number of member variables
...
+};
 
 /*
  2)
  */
+struct Dog
+{
+    char* name, breed;
```
Comment (matkatmusic) [inline main.cpp]: too soon for pointers.
- [ ] Keep It Simple for now.

File: main.cpp [hunk 3]
```diff
@@ -53,42 +53,152 @@ struct CarWash            //1) a U.D.T. with a random number of member variables
...
+    void climbTree( float howHigh );
+    
+}; 
 
 /*
  5)
  */
+struct GuitarPedal
+{
+    char* nameOfPedal;
```
Comment (matkatmusic) [inline main.cpp]: too soon for pointers.   use `std::string` instead. 
- [ ] Keep It Simple for now.

File: main.cpp [hunk 4]
```diff
@@ -53,42 +53,152 @@ struct CarWash            //1) a U.D.T. with a random number of member variables
...
+    int numOfKnobs;
+    bool isModulation;
+
+    void setPedalType( int pedalType );
+    char* getPedalName( GuitarPedal pedal );
```
Comment (matkatmusic) [inline main.cpp]: too soon for pointers.   use `std::string` instead. 
- [ ] Keep It Simple for now.

File: main.cpp (hunk 1 above)
Comment (colliderman) [reply main.cpp]: My bad. I'm already pretty familiar with this stuff so far, so I was just trying to spice it up. Removed the pointers.

**Pull request merged**